Packs/FeedDHS/Integrations/DHS_Feed/DHS_Feed_test.py

```diff
@@ -167,13 +159,13 @@ def test_get_none_existing_path_with_prefix(self, dict_data, path):
         output = safe_data_get(dict_data, path, prefix='TEST')
         assert output is None
 
-    data_test_get_none_existing_path_with_prefix = [
+    data_test_get_none_existing_path_with_prefix2 = [
```

Comment:
why the rename?

Reply:
ruff wasnt happy that it was overriding another variable